app/controllers/shelters_controller.rb

```diff
@@ -26,9 +27,14 @@ def edit
 
   def update
     @shelter = Shelter.find(params[:id])
-    @shelter.update(shelter_params)
 
+   if  @shelter.update(shelter_params)
     redirect_to "/shelters/#{@shelter.id}"
+   else
+    flash[:notice] = @shelter.errors.full_messages.join(". ").to_s
```

Comment:
This will print "column_name can't be blank" to the screen for each missing field